README.md

```diff
@@ -121,17 +98,18 @@ To install HDP Search, follow the [HDP Search instructions](https://docs.hortonw
     ambari-server restart
     ```
 
-### 3. Setup DSX Local
+Please check the Solr is up and running, and the version is at 6.6.2. You can go the Ambari UI, go to Solr session, click the `Quick Links`, then click the `Solr UI`, it will go Solr's instance, it will show the versions. 
```

Comment:
Can you please restore what i had for "how to verify solr is up and running"... just take out the inlining. The rest looked good to me.

Reply:
this is how it was ..
To install HDP Search, follow the HDP Search instructions. This code pattern was tested with Solar v6.6.2 and assumes that Solr is started in cloud mode. Minimally, this can be accomplished by performing the following steps:

After installing HDP Search (Solr), verify that the server is running by performing one of the two methods.

Access the Solr admin dashboard:

http://solr_host:8983/solr
OR

Run the solr status command:

cd <solr_installation_dir>/solr/bin
./solr status
The last step is to install the Solr vector scoring plugin. Please follow the installation instructions on the page to install the plugin.

Finally, restart the Ambari server one last time.

ambari-server restart
Please check the Solr is up and running, and the version is at 6.6.2. You can go the Ambari UI, go to Solr session, click the Quick Links, then click the Solr UI, it will go Solr's instance, it will show the versions.

Reply:
i don't see anything amiss in this. Please restore and add only if anything is amiss.

Reply:
done